widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -541,7 +541,7 @@ table#email-schedule-panel tr:nth-child(2) input {
 }
 
 div#reportControlPanel select {
-  margin: 0;
+  margin: 5px;
```

Comment:
I can only find the element `reportControlPanel` in the context of dashboard printing. Does not seem related to this story. Also, what's the goal of the change, focus ring being clipped?

Reply:
Just found it in PIR as well.
Please use `var(--focus-ring-padding)` instead of the constant `5px` if the goal is to fully display the focus ring.

Reply:
![image](https://github.com/pentaho/pentaho-commons-gwt-modules/assets/113503432/c1d9b3ce-bf44-4bb1-8766-98f9117411ca)
The change was meant to fix the focus ring being clipped here.